doc/en/user/source/data/webadmin/layergroups.rst

```diff
@@ -9,9 +9,10 @@ A layer group also provides a consistent, fixed ordering of the layers it contai
 Layer Group modes
 -----------------
 
-Layer group behaviour can be configured by setting its :guilabel:`mode`. There are 4 available values:
+Layer group behaviour can be configured by setting its :guilabel:`mode`. There are 5 available values:
 
-* **single**: the layer group is exposed as a single layer with a name.
+* **single**: the layer group is exposed as a single layer with a name, acting as an alias for a list of layers. The layers are still showing up as top level entries in the WMS capabilities document (unless explicitly referred by a tree group).
```

Comment:
If a layer is also accessible via a named, EO, or container group it's going to be accessible anyways. Same as with the previous group security logic, a layer is accessible if there is at least a non single group that makes it so.